Misc/NEWS

```diff
@@ -1046,6 +1046,8 @@ Core and Builtins
 Library
 -------
 
+- bpo-28334: fix netrc not working when $HOME is not set
```

Comment:
Uppercase please and colon at end. And instead of fixing, I'd rather something like "Use os.path.expanduser rather than barely checking $HOME is set or not when no argument is given to netrc.netrc().". Ignore my wording, I am not a native speaker. :-(